app/services/coronavirus_pages/content_builder.rb

```diff
@@ -71,6 +72,10 @@ def github_data
     @github_data ||= github_raw_data["content"]
   end
 
+  def github_live_stream_data
+    github_data["live_stream"] ||= {}
```

Comment:
Is this supposed to be just `||`?

Reply:
Yes, thank you. Have amended the commit to remove it.